### PR TITLE
Bugfix/prevent s how network error

### PR DESCRIPTION
**Pull request by mlobac**
## What does this PR do?

Network popup often open fix
[GW][iOS] Network mismatch error dissappers too fast without user interaction fix
Cover app with splash screen when app in multitasking mode to add more security

- [+ ] Checked on iOS
- [ +] Checked on Android

### CODE REVIEW

File: src/state/electrumX/sagas.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@ export function* checkConnection() {
   } catch (e) {
     logger.error('electrumX sagas', `checkConnection error: ${e.message}`);
   } finally {
-    RNBootSplash.hide({ fade: true });
+    RNBootSplash.hide();
```
Comment (panradadev) [inline src/state/electrumX/sagas.ts:224]: with this, we little change behaviour splash it makes sense?

File: src/state/electrumX/sagas.ts [hunk 2]
```diff
@@ -241,8 +241,11 @@ export function* listenToInternetConnection() {
   const chan: unknown = yield call(emitInternetConnectionChange);
 
   while (true) {
+    yield delay(2000); // In case connection with electrum lost we should wait to reconnect before showing alert;
```
Comment (panradadev) [inline src/state/electrumX/sagas.ts:244]: little not feel good with this type solution

File: src/state/electrumX/sagas.ts (hunk 1 above)
Comment (mlobac) [reply src/state/electrumX/sagas.ts:224]: it was me who did this change previously :)

File: src/state/electrumX/sagas.ts (hunk 2 above)
Comment (mlobac) [reply src/state/electrumX/sagas.ts]: fixed

**Review by panradadev — APPROVED** [review]

**Pull request merged**